src/test/unit/controllers/SignInController.test.ts

```diff
@@ -0,0 +1,87 @@
+import sinon from 'sinon';
+import { Response } from 'express';
+import {mockRequest} from '../mocks/mockRequest';
+import SignInController from '../../../main/controllers/SignInController';
+
+const signInController = new SignInController();
+describe('Sign In Option Controller', () => {
+  let i18n = {};
+  it('should render the search options page', () => {
+
+    i18n = {
```

Comment:
this can be set on line 8, doesn't seem like you're reassigning i18n

Reply:
done